src/utils/githook.ts

```diff
@@ -154,21 +184,31 @@ export function processGithook(
       return ''
     }
   }
-  if ('head_commit' in event) {
-    return pushAction(event)
-  } else if ('ref_type' in event) {
-    return createAction(event)
+  if (event_name === 'push') {
+    return pushAction(event as PushEvent)
+  }
+  if (event_name === 'create') {
+    return createAction(event as CreateEvent)
+  }
+  if (event_name === 'delete') {
+    return deleteAction(event as DeleteEvent)
   }
-  for (const prop of props) {
-    if (prop in event) {
-      if ('action' in event) {
-        if (actionsMap[prop]) {
-          if (actionsMap[prop][event.action]) {
-            return actionsMap[prop][event.action](event)
-          }
-        }
+  if ('action' in event) {
+    if (actionsMap[event_name]) {
+      if (actionsMap[event_name][event.action]) {
+        return actionsMap[event_name][event.action](event)
       }
     }
   }
   return ''
 }
+
+function trunc(str: string) {
+  return truncateString(str, MAX_MSG_LENGTH)
+}
+function truncateString(str: string, num: number) {
```

Comment:
are we able to have one function with an optional param set as `MAX_MSG_LENGHT` instead of two functions?

Reply:
i think its fine like this :) because possibly there will be another different truncate with a different length at some point later on?